### PR TITLE
Document the decision making policy.

### LINKED ISSUE #27: Trustee Decision Making Process
Now that we have adopted this repository for recording trustee decisions, we should record the exact process we are using in a policy document.

### LINKED ISSUE #82: Document label usage
@meatballs I see you've added some labels here - could you document their intended usage please?

E.g. I see there are issues tagged with "Trustee Meeting Decision" ("Requires approval from a majority of trustees present at the meeting") but they don't (yet) appear to refer to any specific meeting

I *think* the intent here is that you're proposing we address this decision at a Trustee meeting (rather than one of the other methods we have of deciding things) but it's not clear - I'd love it if you could clarify please

### PR DESCRIPTION
**Pull request by drvinceknight**
Closes #27
Closes #82

### CODE REVIEW

**Review by meatballs — APPROVED** [review]

File: source/policies/decision-making.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Agreement of decisions
+**********************
+
+For a decision to be agreed the following has to occur:
+
+1. A pull request on the `ukpa internaldocs github repository
+   <https://github.com/UKPythonAssociation/ukpa-internaldocs>`_ must be opened
+   with the modifications recorded in a :code:`source/decisions/<file.rst>`.
+2. This pull request should be labeled with either :code:`Trustee Meeting
+   Decision` or :code:`Resolution Decision` as appropriate.
```
Comment (rebkwok) [inline source/policies/decision-making.rst:29]: Should these be defined (or linked to, if it's defined somewhere else already)?  Is a Resolution Decision just any decision that doesn't happen at a Trustee meeting, or is there something more to it?

File: source/policies/decision-making.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Meeting records
+---------------
+
+All meetings of the UKPA are recorded in an :code:`rst` file in the
+:code:`ukpa-internaldocs/source/meetings/` subdirectory:
+
+- General meetings are recorded in
+  :code:`ukpa-internaldocs/source/meeting/general/`
+- Trustee meetings are recorded in
+  :code:`ukpa-internaldocs/source/meeting/general/`
```
Comment (rebkwok) [inline source/policies/decision-making.rst:54]: meeting -> meetings in the two links above

**Review by rebkwok — COMMENTED** [review]
One question and some comments on typos

File: source/policies/decision-making.rst (hunk 1 above)
Comment (meatballs) [reply source/policies/decision-making.rst:29]: Section 17 of our constitution defines two ways in which trustees may make decisions - at a meeting or by resolution  - and they have different requirements for us to meet in order for them to be valid.

**Review by rebkwok — APPROVED** [review]
:+1: pending typos

**Comment by AnnJBarr** [comment]
Subject to typos picked up by @rebkwok LGTM

**Review by AnnJBarr — APPROVED** [review]
Subject to typos picked up by Becky LGTM

**Comment by meatballs** [comment]
Merging as now has majority approval

**Pull request merged**